vta/hardware/chisel/Makefile

```diff
@@ -15,5 +15,81 @@
 # specific language governing permissions and limitations
 # under the License.
 
+CONFIG = DefaultF1Config
+TOP = VTA
+TOP_TEST = Test
+BUILD_NAME = build
+USE_TRACE = 0
+VTA_LIBNAME = libvta_hw
+
+config_test = $(TOP_TEST)$(CONFIG)
+vta_dir = $(abspath ../../)
+tvm_dir = $(abspath ../../../)
+verilator_inc_dir = /usr/local/share/verilator/include
```

Comment:
You mean like having the variable in capital letters and via `export`? normally is installed in that path both in Linux and via homebrew.

Reply:
Yes, that's what I meant - I think it's better to rely on an environment variable via `export` and make that explicit in the instructions. We already instruct to modify the env variable for `PYTHON_PATH` so this will future proof against path changes.